openhtf/io/output/mfg_inspector.py

```diff
@@ -430,18 +447,30 @@ def UploadTestRun(testrun, destination, credentials=None):
         raise UploadFailedError(resp, content)
 
   def __call__(self, test_record):  # pylint: disable=invalid-name
-    credentials = oauth2client.client.SignedJwtAssertionCredentials(
-        service_account_name=self.user,
-        private_key=self.keydata,
-        scope=self.SCOPE_CODE_URI,
-        user_agent='OpenHTF Guzzle Upload Client',
-        token_uri=self.token_uri)
-    credentials.set_store(self._MemStorage())
-
     testrun = _TestRunFromTestRecord(test_record)
+
     try:
+      credentials = oauth2client.client.SignedJwtAssertionCredentials(
+          service_account_name=self.user,
+          private_key=self.keydata,
+          scope=self.SCOPE_CODE_URI,
+          user_agent='OpenHTF Guzzle Upload Client',
+          token_uri=self.token_uri)
+      credentials.set_store(self._MemStorage())
       self.UploadTestRun(testrun, self.DESTINATION_URL, credentials)
-    except UploadFailedError:
+    except:
```

Comment:
Use `except Exception` if you want it to catch all the errors, not a bare except.